src/Sarif.Multitool.FunctionalTests/Update-Expected.ps1

```diff
@@ -1,28 +1,38 @@
 ﻿<#
 .SYNOPSIS
-Update the expected output files for the SarifCli functional tests.
+Update the expected output files for the Sarif.Multitool functional tests.
 
 .DESCRIPTION
-This script runs "SarifCli validate" on each input log file in the SarifCli
+This script runs "Sarif.Multitool validate" on each input log file in the SarifCli
 functional tests TestData directory. For each file foo.sarif, it produces
 an output file foo_Expected.sarif, overwriting any existing file of that
 name in the TestData directory.
 
-.PARAMETERS
-    Configuration
-        The build configuration. Default: Debug
+.PARAMETER Configuration
+    The configuration of the build that produced Sarif.Multitool.exe.
+    Default: Debug
```

Comment:
>Debug [](start = 13, length = 5)

is this the right default? we typically would want to run this script against a release build wouldn't we?

Reply:
I can certainly change it. It doesn't much matter because the Debug and Release builds will of course produce the same SARIF file. My reasoning (I don't remember if I wrote this script, but I mean, my reasoning as I was re-reading and modifying it), was that during development you're more likely to have a Debug build in hand, and it's usually only your run of BuildAndTest.cmd that produces a Release build. I'd like to leave as is, but feel free to push back.

---
In reply to: [205215433](https://github.com/Microsoft/sarif-sdk/pull/968#discussion_r205215433) [](ancestors = 205215433)